src/main/java/no/nav/tag/tiltaksgjennomforingprosess/journalpost/factory/JournalpostFactory.java

```diff
@@ -23,26 +29,26 @@ public Journalpost konverterTilJournalpost(Avtale avtale) {
         journalpost.setBruker(bruker);
 
         //TODO Sette i gang PDF konvertering av avtale her
-        //TODO Sette i gang XML konvertering av avtale her
+
+        String dokumentXml = avtaleTilXml.genererXml(avtale);
 
         Dokument dokument = new Dokument();
         dokument.setDokumentVarianter(Arrays.asList(
-                new DokumentVariant("PDF", encodeToBase64(avtale, true)),
-                new DokumentVariant("XML", encodeToBase64(avtale, false)))
+                new DokumentVariant("PDF", encodeToBase64(dokumentXml, false)),
```

Comment:
```suggestion
                new DokumentVariant("XML", encodeToBase64(dokumentXml, false)),
```
Ser litt motsatt med den andre linjen!!!